react-validated-input-component/src/validated-input.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+class ValidatedInput extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      password: ''
+    };
+
+    this.tooLittleChar = this.tooLittleChar.bind(this);
+  }
+
+  tooLittleChar(event) {
+    this.setState({ password: event.target.value });
+    const password = event.target.value;
+    if (password.length < 7) {
+      this.setState({
+        password: event.target.value
+      });
+    }
```

Comment:
This code doesn't seem to really be doing anything. Here you only need the `setState` call to update `password` in state, nothing else is needed in this method.